test/snapp_auction.js

```diff
@@ -612,6 +613,16 @@ contract("SnappAuction", async (accounts) => {
       assert.equal(auctionIndex, 1)
     })
 
+    it("Large sell order", async () => {
+      const instance = await SnappAuction.new()
+      await setupEnvironment(MintableERC20, instance, token_owner, [user_1], 2)
+      const largeNumberString = "79228162514264333000000000000"  // compare with 2**96 = 7.922816251426434e+28
```

Comment:
Shouldn't this be actually 2**96? (`79228162514264337593543950336`)

Reply:
Please refer to the comment in the description.

Reply:
So why not use `new BN(2).pow(new BN(96))` so that this also serves as documentation for someone looking to use this?

Reply:
Awesome thanks, worked like a charm.